tsconfig.json

```diff
@@ -26,6 +26,11 @@
     "./src/**/*",
     "./src/**/*.json",
     "./tests/**/*",
+    "./scripts/**/*",
     "./benches/**/*"
-  ]
+  ],
+  "ts-node": {
+    "require": ["tsconfig-paths/register"],
+    "transpileOnly": true
+  }
```

Comment:
Going to find out why `swc` isn't working. But this should be mergeable now. I'll update it accordingly when I figure it out.

Reply:
I've added swc back here and used `new this()` instead of `new X()` for the decorated classes here (`iNodeManager` and `EncryptedFS`). This is a quick fix that was discovered here https://github.com/TypeStrong/ts-node/issues/1844#issuecomment-1193723878